aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -110,15 +90,47 @@ The tutorial assumes familarity with VS Code. See [Getting started with VS Code]
 
 * Name the project **MvcMovie**, and then select **Create**.
 
-### Launch the app
+---  
+<!-- End of VS tabs -->
+
+### Run the app
+
+# [Visual Studio](#tab/visual-studio) 
+
+Select **Ctrl-F5** to run the app in non-debug mode.
+
+[!INCLUDE[](~/includes/trustCertVS.md)]
+
+* Visual Studio starts [IIS Express](/iis/extensions/introduction-to-iis-express/iis-express-overview) and runs your app. Notice that the address bar shows `localhost:port#` and not something like `example.com`. That's because `localhost` is the standard hostname for your local computer. When Visual Studio creates a web project, a random port is used for the web server. In the image above, the port number is 5000. The URL in the browser shows `localhost:5000`. When you run the app, you'll see a different port number.
+* Launching the app with **Ctrl+F5** (non-debug mode) allows you to make code changes, save the file, refresh the browser, and see the code changes. Many developers prefer to use non-debug mode to quickly launch the app and view changes.
+* You can launch the app in debug or non-debug mode from the **Debug** menu item:
+
+![Debug menu](start-mvc/_static/debug_menu.png)
+
+* You can debug the app by selecting the **IIS Express** button
+
+![IIS Express](start-mvc/_static/iis_express.png)
+
+# [Visual Studio Code](#tab/visual-studio-code) 
+
+* Press **Ctrl-F5** to run without the debugger.
```

Comment:
This bullet is out of sync with the non-bulleted instructions in the include.  Suggest you either remove the bullet here or put the instructions in the include in bullets.

Reply:
Keyboard instructions should be plain text (Ctrl+F5, rather than **Ctrl-F5**).